src/components/App.js

```diff
@@ -11,15 +11,18 @@ class App extends Component {
     this.state = {
       total: null,
       next: null,
-      // eslint-disable-next-line react/no-unused-state
       operation: null,
     };
     this.handleClick = this.handleClick.bind(this);
   }
 
-  handleClick(buttonName) {
-    const oldState = this.state;
-    this.setState(calculate(oldState, buttonName));
+  handleClick = (name) => {
```

Comment:
- [ ] Thank you for the explanation but you should implement that change As requested by previous CR.

Reply:
Oh I get it, I was somehow just blocked from seeing the changes I needed to make.

Reply:
handleClick = (name) => {
    const { total, next, operation } = calculate(this.state, name);
    this.setState({
      total,
      next,
      operation,
    });
  }
  
  That was already implemented, not sure how I can change it again.